hie.yaml

```diff
@@ -1,13 +0,0 @@
-cradle:
```

Comment:
Only to be sure, did you delete the hie.yaml on purpose?

Reply:
Yes, now that implicit cradles are used in hls this repos hie.yaml file is redundant.